gradle/libs.versions.toml

```diff
@@ -8,8 +8,8 @@ atomicfu = "0.26.1"
 # Sample versions
 androidx-compose = "2023.10.01"
 androidx-fragment = "1.6.2"
-ksp = "2.1.0-1.0.29"
-nativecoroutines = "1.0.0-ALPHA-38"
+ksp = "2.1.20-Beta1-1.0.29"
+nativecoroutines = "1.0.0-ALPHA-38-kotlin-2.1.20-Beta1"
```

Comment:
Thanks for upgrading the versions. Are we sure kotlin 2.1.20-Beta1 is stable and safe to use ?